src/Graphics/Canvas.purs

```diff
@@ -121,10 +118,7 @@ foreign import data CanvasElement :: *
 foreign import data Context2D :: *
 
 -- | An image data object, used to store raster data outside the canvas.
-foreign import data ImageData :: *
-
--- | An array of pixel data.
-foreign import data CanvasPixelArray :: *
+type ImageData = { width :: Number, height :: Number, data :: Uint8ClampedArray }
```

Comment:
I haven't worked with the FFI before. The underlying value the canvas gives us will be a Number with an integral value. If I can just declare it as Int here and that works, that's great. If I need to explicitly call `fromNumber` on it inside the js function I'm happy to do that too.